examples/fine-tuned-RAG/ModelFinetune.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<p>This chart is worth about $20. The outputs have been left inline, because that is the convention that OpenAI is following. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/openai-cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>